### PR TITLE
Add new sthlm delivery readmes

### PR DESCRIPTION
**Pull request by aanil**
Adding the new Delivery Readmes which will be used by Sthlm node.

### CODE REVIEW

**Review by sofiahaglund — APPROVED** [review]
👍

File: delivery_readmes/DELIVERY.README.MethylSeq.txt [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+|--ProjectID/
+   |--00-Reports/
+    --ProjectName_lanes_info.txt
+    --ProjectName_library_info.txt
+    --ProjectName_multiqc_report.html
+    --ProjectName_project_summary.html
+    --ProjectName_project_summary.md
+    --ProjectName_sample_info.txt
+    |--manifestFiles/
+           --<NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txt
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.MethylSeq.txt:48]: This file representation looks a little odd.. Could we do the normal tree output instead? eg:

```suggestion
── ProjectID
   ├── 00-Reports/
   ├── ProjectName_lanes_info.txt
   ├── ProjectName_library_info.txt
   ├── ProjectName_multiqc_report.html
   ├── ProjectName_project_summary.html
   ├── ProjectName_project_summary.md
   ├── ProjectName_sample_info.txt
   └── manifestFiles/
       └── <NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txt
```

Also for the other file representations below..

Comment (aanil) [reply delivery_readmes/DELIVERY.README.MethylSeq.txt]: Done!

File: delivery_readmes/DELIVERY.README.MethylSeq.txt [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+--<NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
+A file summarizing the run with information about instruments and library.
+==================================================================
+ProjectID -> SampleN
+==================================================================
+Each sample directory will contain the following subfolders:
+
+── ProjectID
+   ├── SampleN/
+    ├── 02-FASTQ/
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.MethylSeq.txt:77]:
```suggestion
── ProjectID/
   └── SampleN/
       └── 02-FASTQ/
```

File: delivery_readmes/DELIVERY.README.MethylSeq.txt [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+── ProjectID
+   ├── 00-Reports/
+      ├── ProjectName_lanes_info.txt
+      ├── ProjectName_library_info.txt
+      ├── ProjectName_multiqc_report.html
+      ├── ProjectName_project_summary.html
+      ├── ProjectName_project_summary.md
+      └── ProjectName_sample_info.txt
+   └── manifestFiles/
+      └── <NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txt
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.MethylSeq.txt:48]:
```suggestion
── ProjectID/
   ├── 00-Reports/
   │   ├── ProjectName_lanes_info.txt
   │   ├── ProjectName_library_info.txt
   │   ├── ProjectName_multiqc_report.html
   │   ├── ProjectName_project_summary.html
   │   ├── ProjectName_project_summary.md
   │   └── ProjectName_sample_info.txt
   └── manifestFiles/
       └── <NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txt
```

File: delivery_readmes/DELIVERY.README.MethylSeq.txt [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    ├── 02-FASTQ/
+
+==================================================================
+ProjectID -> SampleN -> 02-FASTQ
+==================================================================
+Contains a subfolder for each flowcell, which in turn contains the FastQ files for each read.
+
+── 02-FASTQ
+    ├── <flowcell>/
+        └── <NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.MethylSeq.txt:86]:
```suggestion
── 02-FASTQ/
   └── <flowcell>/
       └── <NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
```

File: delivery_readmes/DELIVERY.README.RAW_DATA.txt [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+── ProjectID
+   ├── 00-Reports/
+           ├── ProjectName_lanes_info.txt
+           ├── ProjectName_library_info.txt
+           ├── ProjectName_multiqc_report.html
+           ├── ProjectName_project_summary.html
+           ├── ProjectName_project_summary.md
+           └── ProjectName_sample_info.txt
+    ├── manifestFiles/
+           └── <NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txt
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.RAW_DATA.txt:48]:
```suggestion
── ProjectID/
   ├── 00-Reports/
   │   ├── ProjectName_lanes_info.txt
   │   ├── ProjectName_library_info.txt
   │   ├── ProjectName_multiqc_report.html
   │   ├── ProjectName_project_summary.html
   │   ├── ProjectName_project_summary.md
   │   └── ProjectName_sample_info.txt
   └── manifestFiles/
       └── <NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txta 
```

File: delivery_readmes/DELIVERY.README.RAW_DATA.txt [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+--<NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
+A file summarizing the run with information about instruments and library.
+==================================================================
+ProjectID -> SampleN
+==================================================================
+Each sample directory will contain the following subfolders:
+
+── ProjectID
+   ├── SampleN/
+    ├── 02-FASTQ/
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.RAW_DATA.txt:77]:
```suggestion
── ProjectID/
   └── SampleN/
       └── 02-FASTQ/
```

File: delivery_readmes/DELIVERY.README.RAW_DATA.txt [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+==================================================================
+ProjectID -> SampleN -> 02-FASTQ
+==================================================================
+Contains a subfolder for each flowcell, which in turn contains the FastQ files for each read.
+
+── 02-FASTQ
+    ├── <flowcell>/
+           └── <NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.RAW_DATA.txt:86]:
```suggestion
── 02-FASTQ/
   └── <flowcell>/
       └── <NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
```

**Review by ewels — COMMENTED** [review]
I fixed it for you 🤓

File: delivery_readmes/DELIVERY.README.MethylSeq.txt [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+--miscellaneous.md5
+A file to check the integrity of the miscellaneous.lst file.
+
+--SampleN.lst
+A list of the raw data files of SampleN.
+
+--SampleN.md5
+A file to check the integrity of the SampleN.lst file.
+File integrity checks
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.MethylSeq.txt:26]:
```suggestion

File integrity checks
```

File: delivery_readmes/DELIVERY.README.MethylSeq.txt [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+A report summarising the information about the run, samples, and library.
+
+--ProjectName_project_summary.md
+The .md file of the project_summary.html file used to generate the report.
+
+ --ProjectName_sample_info.txt
+A file with information about the samples such as names, million reads, and Q30 values.
+--<NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
+A file summarizing the run with information about instruments and library.
+==================================================================
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.MethylSeq.txt:70]:
```suggestion

--<NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
A file summarizing the run with information about instruments and library.


==================================================================
```

File: delivery_readmes/DELIVERY.README.RAW_DATA.txt [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+--miscellaneous.md5
+A file to check the integrity of the miscellaneous.lst file.
+
+--SampleN.lst
+A list of the raw data files of SampleN.
+
+--SampleN.md5
+A file to check the integrity of the SampleN.lst file.
+File integrity checks
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.RAW_DATA.txt:26]:
```suggestion

File integrity checks
```

File: delivery_readmes/DELIVERY.README.RAW_DATA.txt [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+A report summarising the information about the run, samples, and library.
+
+--ProjectName_project_summary.md
+The .md file of the project_summary.html file used to generate the report.
+
+ --ProjectName_sample_info.txt
+A file with information about the samples such as names, million reads, and Q30 values.
+--<NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
+A file summarizing the run with information about instruments and library.
+==================================================================
```
Comment (ewels) [inline delivery_readmes/DELIVERY.README.RAW_DATA.txt:70]:
```suggestion
A file with information about the samples such as names, million reads, and Q30 values.

--<NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
A file summarizing the run with information about instruments and library.


==================================================================
```

**Review by ewels — COMMENTED** [review]
Sorry, some more minor whitespace stuff, now I'm done.. 😅

**Comment by ewels** [comment]
beautiful

**Review by ewels — APPROVED** [review]

**Comment by aanil** [comment]
Merging with approvals.

**Pull request merged**